### ISSUE TITLE
Opening file tree from plugin causes crash

### ISSUE THREAD

**Issue by uninstall-your-browser** [curlpipe/ox#188]
**What is the bug?**
Loading a plugin that opens the file tree on startup causes a crash

The plugin:
```lua
editor:toggle_file_tree()
```
(just that one line)

The error
```
panicked at src/editor/interface.rs:261:49:
called `Option::unwrap()` on a `None` value
Return 101
```

**What did you do to get the bug?**
Load the plugin

**What behaviour were you expecting?**
Not crashing

**Screenshots (if applicable)**

**Comment by curlpipe** [comment]
Thanks for the bug report, I think I know what is causing this and it should be fixed in the next release.

**Comment by curlpipe** [comment]
Should be fixed in 0.7.5, which was just released

Let me know if you're still encountering issues, thanks!
